spock-specs/src/test/groovy/org/spockframework/smoke/ast/AstSpec.groovy

```diff
@@ -44,6 +43,22 @@ class AstSpec extends EmbeddedSpecification {
     snapshotter.assertThat(result.source).matchesSnapshot()
   }
 
+  def "foo"() {
+    given:
+    snapshotter.featureBody()
+
+    when:
+    def result = compiler.transpileFeatureBody('''
+    given:
+    MockStatic(StaticClass) { sc ->
+      sc.staticMethod() >> "MockValue"
+    }
```

Comment:
This API does not exist.
I guess the whole test was just a quick test to our discussion.

Reply:
Yeah...